config/owasp/suppressions.xml

```diff
@@ -82,4 +82,9 @@
       Wait for spring boot upgrades</notes>
     <cve>CVE-2023-35116</cve>
   </suppress>
+  <suppress until ="2023-12-30">
+    <notes>False positive: Jackson community does not consider this a security issue (https://github.com/FasterXML/jackson-databind/issues/3972)
+      Wait for spring boot upgrades</notes>
+    <cve>CVE-2023-33202</cve>
```

Comment:
Should we consider moving this out to 2024-01-31? Similar to the recent dates we used for other cve's.